### ISSUE TITLE
New jetty versions released

### ISSUE THREAD

**Issue by victornoel** [jirutka/jetty-bom#3]
Hi,

Happy new year :)

Jetty 9.2.20 (no changes since previous), 9.3.15 (no changes since previous) and 9.4.0 (see below) were released!

The changes between 9.3.15 and 9.4.0 are the following:
```
+org.eclipse.jetty.cdi:cdi-core
+org.eclipse.jetty.cdi:cdi-full-servlet
+org.eclipse.jetty.cdi:cdi-servlet
+org.eclipse.jetty.cdi:cdi-websocket
+org.eclipse.jetty.cdi:jetty-cdi-parent
+org.eclipse.jetty.gcloud:gcloud-parent
+org.eclipse.jetty.gcloud:jetty-gcloud-session-manager
+org.eclipse.jetty.http2:http2-alpn-tests
+org.eclipse.jetty.http2:http2-client
+org.eclipse.jetty.http2:http2-common
+org.eclipse.jetty.http2:http2-hpack
+org.eclipse.jetty.http2:http2-http-client-transport
+org.eclipse.jetty.http2:http2-parent
+org.eclipse.jetty.http2:http2-server
-org.eclipse.jetty:jetty-cdi
+org.eclipse.jetty:jetty-documentation
+org.eclipse.jetty:jetty-home
+org.eclipse.jetty:jetty-infinispan
-org.eclipse.jetty:jetty-jsp
-org.eclipse.jetty:jetty-monitor
+org.eclipse.jetty:jetty-unixsocket
+org.eclipse.jetty.memcached:jetty-memcached-sessions
+org.eclipse.jetty.memcached:memcached-parent
-org.eclipse.jetty.spdy:spdy-alpn-tests
-org.eclipse.jetty.spdy:spdy-client
-org.eclipse.jetty.spdy:spdy-core
-org.eclipse.jetty.spdy:spdy-example-webapp
-org.eclipse.jetty.spdy:spdy-http-client-transport
-org.eclipse.jetty.spdy:spdy-http-common
-org.eclipse.jetty.spdy:spdy-http-server
-org.eclipse.jetty.spdy:spdy-parent
-org.eclipse.jetty.spdy:spdy-server
+org.eclipse.jetty:test-jetty-webapp
+org.eclipse.jetty:test-proxy-webapp
```

**Comment by victornoel** [comment]
Hi,

Again some new versions were released :)

By the way, I asked jetty people for including a BOM in their release: eclipse/jetty.project#1348

**Comment by joakime** [comment]
do not include the cdi-* components unless you are actually using a server wide Weld setup (which is awkward)

do not include the gcloud-* components unless you are running on the Google Flex environment.

do not include the jetty-infinispan component unless you are running on an actual infinispan node.

do not include the jetty-unixsocket component as that will only work on limited linux environments (not OSX or Windows)

do not include the http2-* components as they require the alpn-boot.jar that is version specific to the JVM you are running on

do not include jetty-home artifact, as that's specifically designed for Docker implementors and has duplicated class names from other modules with different implementations.

do not include the test-* components, why?

**Comment by victornoel** [comment]
@joakime it does not include anything, it just sets version for these artifacts, see my answer on eclipse/jetty.project#1348.

**Comment by victornoel** [comment]
@jirutka finally, this has been included in jetty, there is BOMs for 9.3.19+ and 9.4.5+ apparently.

Thanks for your work until now :)
